amps/ags/rm-automation/rm-automation-community-rest-api/src/test/java/org/alfresco/rest/rm/community/fileplans/FilePlanTests.java

```diff
@@ -91,6 +98,8 @@ public class FilePlanTests extends BaseRMRestTest
 {
     // ** Number of children (for children creation test) */
     private static final int NUMBER_OF_CHILDREN = 10;
+    /** A list of capabilities. */
+    private static final java.util.Set<String> CAPABILITIES = newHashSet(UserCapabilities.VIEW_RECORDS_CAP, UserCapabilities.DECLARE_RECORDS_CAP);
```

Comment:
I think we don't need the package declaration for `Set`. What is more, we can set this variable as local in the `newRoleUserFilePlanRolesAndCapabilities` method.